pkg/operator/highcpuusagealertcontroller/performanceprofile.go

```diff
@@ -0,0 +1,66 @@
+package highcpuusagealertcontroller
+
+import (
+	"context"
+	"encoding/json"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/utils/cpuset"
+)
+
+// default and taken from the docs
+const defaultCoresNum = 8
+
+var performanceGroup = schema.GroupVersionResource{Group: "performance.openshift.io", Version: "v2", Resource: "performanceprofiles"}
```

Comment:
Will this API/GVK be always available on an SNO platform ?

Why aren’t the performance profiles part of openshift/api?

Reply:
> Why aren’t the performance profiles part of openshift/api?

ok, from the internal discussion we had on Slack, the profile should be part of the openshift/api since it is a public API (used by users). It is a technical debt, and will eventually be moved there.

Reply:
Will the [CRD](https://github.com/openshift/cluster-node-tuning-operator/blob/master/manifests/20-performance-profile.crd.yaml) be installed on an `SNO` cluster when the `cpuPartitioningMode` is set to `AllNodes`. Or could it happen that the API/CRD won't be available at all?

Reply:
When is the `cluster-node-tuning-operator` actually installed on the cluster?

Reply:
Is the operator installed by CVO or manually be end users ?

Reply:
I've found https://github.com/openshift/enhancements/blob/master/enhancements/workload-partitioning/wide-availability-workload-partitioning.md#workflow-description which indicates that the operator and the API/CRD is actually installed/created during cluster creation.

Reply:
@qJkee could you please also create a bug/story for moving `performanceprofiles` to `openshift/api` repo ?